### PR TITLE
adds audit log

### PR DESCRIPTION
**Pull request by workaj**
currently wip

### CODE REVIEW

File: core/actions/Action_contentcreate.php [hunk 1]
```diff
@@ -0,0 +1,105 @@
+<?php
+defined('CMSPATH') or die; // prevent unauthorized access
+
+class Action_contentcreate extends Actions {
+
+    public function display() {
+        $contantTableName = Content::get_table_name_for_content_type($this->options->content_type);
```
Comment (bobmitch) [inline core/actions/Action_contentcreate.php:7]: fix typo

**Review by bobmitch — CHANGES_REQUESTED** [review]
looks good other than spelling errors in variables that should be addressed

**Comment by bobmitch** [comment]
Note: I didn't flag every line where the typo occurred, just the scope

File: core/actions/Action_contentdelete.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?php
+defined('CMSPATH') or die; // prevent unauthorized access
+
+class Action_contentdelete extends Actions {
+
+    public function display() {
+        $contantTableName = Content::get_table_name_for_content_type($this->options->content_type);
```
[the remark this answers is not in the document]
Comment (workaj) [reply core/actions/Action_contentdelete.php]: https://github.com/HoltBosse/Alba/pull/346/commits/be136702ab2b49a1ca8068633d5f57b153852fe6

File: core/actions/Action_contentupdate.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?php
+defined('CMSPATH') or die; // prevent unauthorized access
+
+class Action_contentupdate extends Action_contentcreate {
+
+    public function display() {
+        $contantTableName = Content::get_table_name_for_content_type($this->options->content_type);
```
[the remark this answers is not in the document]
Comment (workaj) [reply core/actions/Action_contentupdate.php]: https://github.com/HoltBosse/Alba/pull/346/commits/be136702ab2b49a1ca8068633d5f57b153852fe6

File: core/actions/Action_contentcreate.php (hunk 1 above)
Comment (workaj) [reply core/actions/Action_contentcreate.php]: https://github.com/HoltBosse/Alba/pull/346/commits/be136702ab2b49a1ca8068633d5f57b153852fe6

**Pull request merged**